composer.json

```diff
@@ -242,7 +242,8 @@
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": false,
             "symfony/flex": true,
-            "symfony/runtime": true
+            "symfony/runtime": true,
+            "symfony/thanks": true
```

Comment:
We did the opposite here https://github.com/Sylius/Sylius-Standard/blob/1.12/composer.json#L72
I don't remember exactly why.

Reply:
Oh, I've no problem with making it false. I just wanted to have it listed to avoid an error. If you want to disable I can change the code.

Reply:
I would vote for the consistency and make it `false` - but definitely, it should be done 👍 Btw, probably should be based on `1.11` branch 🖖

Reply:
Done a new PR here: https://github.com/Sylius/Sylius/pull/14588 because it conflicts with this commit

Reply:
I also changed the base of this PR to make it against 1.12 (no conflicts for this)